websites/C/Comic Space/dist/metadata.json

```diff
@@ -0,0 +1,19 @@
+{
+    "author": {
+       "name": "Nanashin",
+       "id": "330030557457022986"
+     },
+     "service": "Comic Space" ,
+     "description": {
+       "en": "One scanlator with a website clean to read online.",
+       "pt_BR": "Uma scanlator com um site limpo para poder ler online.",
+       "pt": "Uma scanlator com um site limpo para poder ler online."
+     },
+     "url": "www.comicspace.com.br",
+     "version": "1.0.0",
+     "logo": "https://i.imgur.com/5uHI3Mm.png",

```

Comment:
logo looks good, but it has some little mistakes that you can easily fix it

Reply:
if you don't have the patience to do those manually, you can easily use a service like [remove.bg](https://remove.bg/)